manage/manage.css

```diff
@@ -259,17 +260,32 @@ select {
 }
 
 /* collapsibles */
+#header details,
+p.border-bottom {
+  border-bottom: 1px solid rgba(204, 204, 204, 0.4);
```

Comment:
such borders aren't helpful, it looks too cramped there, but I wouldn't want to just add space so that borders can breathe, so let's remove them - it's not like the blocks are rows in a table